### DISCUSSION TITLE
Trimming output in development runs

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#70] [TF4: Diagnostics, output]
We talked about trimming the output in development runs. 

I started a [googledocs](https://docs.google.com/document/d/1pMOAJ_tPL48mseFfiDaT537NwRumchM-kCp67QGLOfk/edit?usp=sharing) where you can add the fields your want to output.

For reference, the current output was:
```
monthly:
fincl1 = 'U','V','Q','T','PSL','OMEGA','PS','PRECC','PRECL','UTGWORO','VTGWORO',
         'TS', 'TAUX','TAUY','TAUBLJX','TAUBLJY','TAUGWX','TAUGWY','FLNT','FLNS','FSNS','FSNT',
         'LHFLX','SHFLX','TMQ','FLDS','FSDS','FSDSC','SWCF','LWCF','PRECSC','PRECSL',  'DTCOND','Z3', 'PSL'
         'PBLH','ZMDT', 'ZMDQ', 'STEND_CLUBB','RVMTEND_CLUBB',
         'ZMDQ','CMFMC_DP','DPDLFLIQ','DPDLFICE','PCLDTOP','CLDTOP', 'KVH_CLUBB',
         'DTCORE','DQCORE','PTEQ','DCQ','QTGW','CT_H2O','DCQ','MPDQ'


daily:
fincl2 = 'TROP_T', 'TROP_P', 'Q', 'T','U850','V850','U200','V200','PRECT','PRECC','PRECL','PBLH',
         'OMEGA500','FLUT','Z500','T500','PS','PMID:I','TREFHTMN:M', 'TREFHTMX:X','TSMN:M', 'TSMX:X',
         'PBLH','ZMDT', 'ZMDQ', 'STEND_CLUBB','RVMTEND_CLUBB',
         'PRECT','PRECL','U200','V200','U850','V850','FLUT','Z500','TREFHT','TS','CLDTOT','SWCF',
         'LWCF','PS','PSL','TMQ','LHFLX','SHFLX','T500','OMEGA500','CLDLOW'


6 hourly:
fincl3 = 'U:I','V:I','T:I', 'OMEGA:I','PS:I','PMID:I', 'OMEGA500','PRECT','U200','U850','FLUT','Q:I', 'ZMDT:I','ZMDQ:I','CMFMC_DP:I','DPDLFLIQ:I','DPDLFICE:I','PCLDTOP:I','CLDTOP:I' ,
       'DTCORE:I','DQCORE:I','PTEQ:I','DCQ:I','QTGW:I','CT_H2O:I','DCQ:I','MPDQ:I'


3 hourly:
fincl4 = 'PRECT','PRECC'
```

and for the h0 file:
```
 clubb_history          =  .false. 
 clubb_rad_history      =  .false. 
 history_aero_optics    =  .false.  
 history_aerosol        =  .false.  
 history_amwg           =  .true.   
 history_budget         =  .false.  
 history_chemistry      = .true.
 history_chemspecies_srf   =       .true.   
 history_clubb          =  .true.   
 history_dust           =  .false.  
 history_eddy           =  .false.  
 history_vdiag          =  .false.  
 history_waccm          =  .true.   
 history_waccmx         =  .false. 
 ```
My own preference would be to start from empty tapes instead of keeping:  ```history_amwg           =  .true.```

**Comment by islasimpson** [comment]
Did you intend to put in a link to the google doc?

**Reply by cecilehannay** [reply]
Yes. I put it in above.

**Reply by islasimpson** [reply]
Oh ok, it wasn't showing up as a link for me before.  But now it is.  Sorry.

**Reply by cecilehannay** [reply]
If it doesn't work:
https://docs.google.com/document/d/1pMOAJ_tPL48mseFfiDaT537NwRumchM-kCp67QGLOfk/edit

**Comment by adamrher** [comment]
It seems that we're coming up w/ a comprehensive h0 list of vars in the google doc, and so headed down the path to set ```empty_htapes=T``` which overwrites any of these ```history_``` settings and so we won't need to worry about how they are being set by default.

**Comment by adamrher** [comment]
Can someone explain why we want QRL QRS at daily? This is the only requested daily variable

**Reply by dan800** [reply]
They are in addition to the daily fields listed above.

**Comment by cecilehannay** [comment]
People could already look at the first run [#78](https://github.com/NCAR/amwg_dev/issues/78) with reduced output and see what they are missing. 

Output is: ```/glade/scratch/hannay/archive/f.e21.FWscHIST.ne30_L48_BL10_cam6_3_041_kzz3_zmtop75.hf.001/atm/hist```


Please add what you need in: [googledocs](https://docs.google.com/document/d/1pMOAJ_tPL48mseFfiDaT537NwRumchM-kCp67QGLOfk/edit?usp=sharing)

@andrewgettelman will look at AOD, AODDUST, etc...

**Reply by andrewgettelman** [reply]
Here is a [google sheet](https://docs.google.com/spreadsheets/d/17IxS8Du8wkLWefzELex3e3RCr3GrpSr6o5D3GZm8xto/edit?usp=sharing)

Please use that going forward. Added some cloud/microphysics fields
